app/definitions/IGetRoomRoles.ts

```diff
@@ -0,0 +1,9 @@
+export interface IGetRoomRoles {
```

Comment:
Should we create this interface inside `definitions/IRoom` or `definitions/IRole` instead create a new file?